superset-frontend/spec/javascripts/sqllab/IconTooltip_spec.jsx

```diff
@@ -18,22 +18,24 @@
  */
 import React from 'react';
 import { shallow } from 'enzyme';
+import { Tooltip } from 'src/common/components/Tooltip';
+import IconTooltip from 'src/components/IconTooltip';
 
-import Link from 'src/components/Link';
-
-describe('Link', () => {
+describe('IconTooltip', () => {
   const mockedProps = {
     tooltip: 'This is a tooltip',
     href: 'https://www.airbnb.com',
```

Comment:
It _looks like_ with the new changes to IconTooltip, this href isn't used anywhere. The tests don't assert against it, so perhaps they should?

Reply:
I'm not sure if the `href` is used in the codebase anywhere... if not, then we're all good. But if it is, then we should find a good way to support it, perhaps rendering a/span conditionally.

Reply:
Actually we need to delete that line from the test because IconTooltip does not support href. If we want to use a `href` we should use a component like `react-router-dom` `Link`.